index.js

```diff
@@ -105,7 +147,16 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
```

Comment:
We forgot to account for a few things here in function, like what if the casing of the genre in the string is different from the genre passed in. It is always important to take a look at what the tests are actually testing.